integrationtest/neo/client/Client.d

```diff
@@ -276,6 +305,63 @@ public class Client
             return succeeded;
         }
 
+        /***********************************************************************
+
+            Assigns a DoublePut request, instructing two nodes to associate the
+            specified value and key. The calling Task is blocked until the
+            request finishes.
```

Comment:
(re: previous comment) - just copy the docs from here.

Reply:
Mm do you mean copy "Assigns a DoublePut request, instructing two nodes to associate the specified value and key." to line 98? The doc is identical.

Reply:
Yes, to copy that, but no, the doc is not identical as far as I can see (the doc at line 98 speaks about singular node).

Reply:
Ahh sorry, yes, you're right.

Reply:
Updated.